main/lsp/test/lsp_file_updates_test.cc

```diff
@@ -0,0 +1,171 @@
+#include "gtest/gtest.h"
+// has to go first as it violates are requirements
```

Comment:
Poisoning specific functions I believe (like strcpy). The comment prevents clang-format from reordering it to be below our poisons -- which would cause a compilation failure.